src/map/ai/controllers/trust_controller.cpp

```diff
@@ -95,14 +102,49 @@ void CTrustController::DoCombatTick(time_point tick)
                 }
             }
         }
+		
+		POwner->PAI->EventHandler.triggerListener("TRUST_COMBAT_TICK", POwner, POwner->PMaster, PTarget);
```

Comment:
This should probably just be COMBAT_TICK - there's not really any reason to have two differently named ones